Sources/VaporAuth/Claims/ScopeClaim.swift

```diff
@@ -42,25 +43,13 @@ extension ScopeClaim: ExpressibleByArrayLiteral {
 
 
 extension ScopeClaim: Comparable {
-    var wrapper: [ScopeWrapper] {
-        self.value.map { ScopeWrapper(raw: $0) }
-    }
-
     public static func < (lhs: ScopeClaim, rhs: ScopeClaim) -> Bool {
-        let requiredWrappers = lhs.wrapper
-        let carriedWrappers = rhs.wrapper
-        
-        for require in requiredWrappers {
-            guard carriedWrappers.contains(where: { require <= $0 }) else  {
-                return false
-            }
-        }
-        return true
+        return Set(lhs.value).isStrictSubset(of: Set(rhs.value))
```

Comment:
[Set.isStrictSubset(of:)](https://developer.apple.com/documentation/swift/set/isstrictsubset(of:)-96vc3) only perform "=="  on each element.
In this method, we required every scope not only equal to one of the carried scopes, but also less than one.
So this part may not replace with set operation.